storage-resize-images/extension.yaml

```diff
@@ -146,6 +146,21 @@ params:
     example: thumbnails
     required: false
 
+  - param: ABSOLUTE_PATH_LIST
+    label: List of absolute paths for resized images
+    description: >
+      A comma-separated list of absolute paths to restrict the 
+      images to be resized. For instance, if you want to resize only 
+      the images stored in `/users/pictures`, then other images uploaded 
+      outside that specific path will not be resized. If you prefer to 
+      resize whole the images uploaded to the bucket, leave this field empty.
```

Comment:
I think reword this part of the description:
"For instance, if you want to resize only the images stored in `/users/pictures`, then other images uploaded outside that specific path will not be resized. If you prefer to resize whole the images uploaded to the bucket, leave this field empty."

To something like this:
"For example, to only resize the images stored in `/users/pictures` directory, specify the path `/users/pictures`. If you prefer to resize every image uploaded to the storage bucket, leave this field empty."

Reply:
Hey @kevinthecheung, do you mind taking a look at this and make sure it sounds ok, please? Thanks.

Reply:
@russellwheatley It's done! :+1: